cfgov/jinja2/v1/_includes/organisms/footer.html

```diff
@@ -55,12 +55,12 @@
                         block__padded-top">
                 {{ social_media.render( { 'is_share_view': false } ) }}
             </div>
-        </div><!-- END .footer-pre -->
+        </div><!-- END .o-footer_pre -->
```

Comment:
The deed is done https://github.com/cfpb/cfgov-refresh/pull/1837/commits/f40911286897766217a845317af9c772f357ae13